src/libs/KeyboardShortcut/index.js

```diff
@@ -177,6 +179,16 @@ function subscribe(key, callback, descriptionKey, modifiers = 'shift', captureOn
     return () => unsubscribe(key);
 }
 
+/**
+ * Return platform specific modifiers for keys like Control (Cmd)
+ * @param {Array} modifiers
+ * @returns {Array}
+ */
+function getShortcutModifiers(modifiers) {
+    const isMacOS = getOperatingSystem() === CONST.OS.MAC_OS;
+    return _.map(modifiers, modifier => (isMacOS && modifier === 'control' ? 'meta' : 'control'));
```

Comment:
Sorry, I know this snippet was from my suggestion but this code is flawed because `['control', 'shift']` on maxOS would become `['meta', 'control']`, which isn't right.

So I think what we really want might be something more like this:

```javascript
// In CONST.js
KEYBOARD_SHORTCUT_MODIFIERS: {
    CTRL: {
        DEFAULT: 'control',
        [PLATFORM.OS.MACOS]: 'meta',
    },
    SHIFT: {
        DEFAULT: 'shift',
    },
},

...
...

// In KeyboardShortcut/index.js
function getShortcutModifiers(modifiers) {
    const operatingSystem = getOperatingSystem();
    return _.map(modifiers, (modifier) => {
        if (!_.has(CONST.KEYBOARD_SHORTCUT_MODIFIERS, modifier)) {
            return modifier;
        }
        
        const platformModifiers = CONST.KEYBOARD_SHORTCUT_MODIFIERS[modifier];
        return lodashGet(platformModifiers, operatingSystem, platformModifiers.DEFAULT || modifier);
    });
}
```